src/Mod/Sketcher/App/PythonConverter.cpp

```diff
@@ -195,6 +195,17 @@ std::string PythonConverter::convert(const std::string& doc,
     return constraintlist;
 }
 
+std::string PythonConverter::makeArryString(const std::stringstream& ss)
+{
+    std::string res = ss.str();
+    // remove last comma and add brackets
+    int index = res.rfind(',');

```

Comment:
Assuming this is going into a python command, a comma at the end is perfectly fine. So `[1.0, 2.0, 3.0,]` is valid.